pebblo/reports/templates/weasyprintTemplate.html

```diff
@@ -273,44 +303,77 @@
           {% endfor %}
         </tbody>
       </table>
+      {% else %}
```

Comment:
We generate report only after loading finishes. so ideally, there won't be a scenario where no data source is present. Either, let's add it to the local ui or remove this empty state from report.

Reply:
Removed data source empty state